user_manual/room-testing-guide.adoc

```diff
@@ -0,0 +1,54 @@
+== Room Testing Procedure ==
+
+Each room needs to be tested for both audio and visual. In-room and recorded levels will be tested.
+
+There are several notes to remember during the each test:
+
+* Almost all settings are adjusted remotely, please be patient.
+* **Any** in-room change shall be as-directed from AV-NOC.
+* Please speak with radio protocol as multiple teams will be working at once
```

Comment:
Please speak according to our radio protocol (see document) as multiple teams...